### DISCUSSION TITLE
Daily Precipitation Potential

### DISCUSSION THREAD

**Discussion by Yeshia** [weather-gov/api#358] [Q&A]
Is there a field for the daily forecast which shows the potential for precipitation? I only see it in the raw hourly gridpoints data. Is there a way to calculate the daily based on the hourly? Thanks!

**Comment by StephenClouse** [comment]
Not at present. This is a frequent request so we have an internal ticket to add this in a future release.

**Reply by robstitt** [reply]
Any update? While this topic and another(s) indicate POP is in the "hourly gridpoints" information, I'm not seeing it there explicitly either (outside of parsing the icon URL). Only a somewhat free-form description (e.g., "Chance Rain Showers"). It would be SO helpful if the percentage POP and the type (showers/rain/heavy rain/snow/blizzard/freezing rain/sleet/etc.) were returned in a couple new fields (e.g., POP, PRECIPTYPE)
